### ISSUE TITLE
Artifact Function definitions

### ISSUE THREAD

**Issue by bdonohue29** [CommonCoreOntology/CommonCoreOntologies#139]
Disclaimer: I didn't see a place to comment on the proposal itself, so I'm opening an issue for feedback/discussion.

In the recent post, Alex proposes defining Artifact Functions with the following format:
"An X Function that inheres in Artifacts (or a specific type of artifact, e.g., Vehicles) that are designed to Y."

I think a problem with this format is that it inadvertently leaves out the tight coupling between function and design. If an artifact can have multiple functions and be designed for multiple purposes, then we need a way of expressing the connection between a particular function and the intent to design it that way (design intent X1 corresponds to function Y1, design intent X2 corresponds to function Y2, etc).

In other words, I think we want to say something like: the X Function inheres in the Artifact _in virtue of_ being designed to Y, not merely that X inheres in the artifact A and A is designed to Y. (I'm tempted to say "happens to be designed to Y".)

**Comment by APCox** [comment]
I agree with this suggestion and have amended the document accordingly.
